Refactor how the team info is displayed

AnnKatF opened the pull request:
**Why?**
This change is to improve our visibility as a team on who we are and what each one of us is working on.

**How?**
- Added a table with the Managers/Leads people we have in the team so everyone is aware of those.
- Merged all the information of each engineer and each squad in just one table.

AnnKatF committed:
Refactored how the team is presented.

AliSoftware commented on README.md:
```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
-By definition, members work on their respective squad, although they are free to work in different squads if the work load justifies it.
+## 1. Who are we? 
 
+==================================
```
Not sure what you expected this to render like, but doesn't seem to render nicely in GitHub.

Maybe you meant to show a horizontal line separator? If so I think it's achieved via `----` in GitHub-Flavored Markdown

AliSoftware commented on README.md:
```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
+| **Chapter Leads** | Ana Catarina Figueiredo • [Twitter](https://twitter.com/AnnKatFig) • [GitHub](https://github.com/AnnKatF)  <br>  David Rodrigues • [Twitter](https://twitter.com/dmcrodrigues) • [GitHub](https://github.com/dmcrodrigues) |
+| **Squad Tech Leads** | Michael Brown • [Twitter](https://twitter.com/mluisbrown) |
 
+==================================
```
ditto

AliSoftware commented on README.md:
```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
+|**Healthcheck** <br> Overview of your health using a 3D body model (avatar). | Ben Henshall • [Twitter](https://twitter.com/ben_henshall?lang=en) • [GitHub](https://github.com/Ben-Henshall) <br> Julien Ducret • [Twitter](https://github.com/brocoo) • [GitHub](https://github.com/brocoo) |
+| **Monitor** <br> Monitoring health metrics like activity, blood, urine, ... | Anders Ha • [Twitter](https://twitter.com/_andersha) • [GitHub](https://github.com/andersio) <br> Daniel Haight • [GitHub](https://github.com/Daniel1of1) <br> Daniel Spindelbauer • [Twitter](https://twitter.com/sdaniel55) • [GitHub](https://github.com/sdaniel55) <br> Diego  Petrucci • [Twitter](https://twitter.com/die ...
+| **Prescriptions** <br> Precriptions functionality. | Adam Borek • [Twitter](https://twitter.com/TheAdamBorek) • [GitHbu](https://github.com/TheAdamBorek) <br> Konrad Muchowicz • [GitHub](https://github.com/konrad-em)
+| **Professional Services** | Simon Cass • [Twitter](https://twitter.com/codercass) • [GitHub](https://github.com/scass91)|
```
No squad description like for the others?

AnnKatF replied:
I still have to know what this is about 😅

AliSoftware commented on README.md:
```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
+|**GP @ Hand** <br> End to end journey for NHS registration. | James Birtwell • [GitHub](https://github.com/jimmybee) |
+|**Healthcheck** <br> Overview of your health using a 3D body model (avatar). | Ben Henshall • [Twitter](https://twitter.com/ben_henshall?lang=en) • [GitHub](https://github.com/Ben-Henshall) <br> Julien Ducret • [Twitter](https://github.com/brocoo) • [GitHub](https://github.com/brocoo) |
+| **Monitor** <br> Monitoring health metrics like activity, blood, urine, ... | Anders Ha • [Twitter](https://twitter.com/_andersha) • [GitHub](https://github.com/andersio) <br> Daniel Haight • [GitHub](https://github.com/Daniel1of1) <br> Daniel Spindelbauer • [Twitter](https://twitter.com/sdaniel55) • [GitHub](https://github.com/sdaniel55) <br> Diego  Petrucci • [Twitter](https://twitter.com/die ...
+| **Prescriptions** <br> Precriptions functionality. | Adam Borek • [Twitter](https://twitter.com/TheAdamBorek) • [GitHbu](https://github.com/TheAdamBorek) <br> Konrad Muchowicz • [GitHub](https://github.com/konrad-em)
```
```suggestion
| **Prescriptions** <br> Precriptions functionality. | Adam Borek • [Twitter](https://twitter.com/TheAdamBorek) • [GitHub](https://github.com/TheAdamBorek) <br> Konrad Muchowicz • [GitHub](https://github.com/konrad-em)
```

AliSoftware commented on README.md:
```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
+| **Prescriptions** <br> Precriptions functionality. | Adam Borek • [Twitter](https://twitter.com/TheAdamBorek) • [GitHbu](https://github.com/TheAdamBorek) <br> Konrad Muchowicz • [GitHub](https://github.com/konrad-em)
+| **Professional Services** | Simon Cass • [Twitter](https://twitter.com/codercass) • [GitHub](https://github.com/scass91)|
+| **SDK** <br> Develop and mantain the SDK frameworks. | Viorel Mihalache • [Twitter](https://twitter.com/viorelMO) • [GitHub](https://github.com/viorel15) |
+| **Tenancy & Features** | Anil Puttabuddhi • [GitHub](https://github.com/anilputtabuddhi) |
```
No squad description like for the others?

AnnKatF replied:
I still have to know what this is about 😅

AliSoftware reviewed (commented):
I like this new table, easier than have to sync 2 tables every time 👍 (Might get trickier to review the diff in future PRs when we need to update to add new people or move people across squads, but 🤷‍♂ )

That means that we don't mention the Slack squad channels anymore though. Nor the new Slack User Group teams recently created. Do you think it's be worth keeping/adding in the table, or do we decide we stop mentioning them if we think they're not useful?

Otherwise, just missing description for 2 squads + fixing typo.

AnnKatF committed:
Update README.md

Co-Authored-By: Olivier Halligon <olivier.halligon@babylonhealth.com>

AliSoftware commented:
@NSMyself doesn't seem to be listed anywhere anymore?

AnnKatF commented:
> Might get trickier to review the diff in future PRs 

Yes I agree that is might be trickier to review but I didn't find any better way to aggregate all the info.

> That means that we don't mention the Slack squad channels anymore though.

To be fair I didn't find those useful. However, if people think they were useful, we can add them just bellow each Squad description, inside the same cell. We can also add in the same place the Slack User Group teams.

AnnKatF commented:
> @NSMyself doesn't seem to be listed anywhere anymore?

His new squad is still to be defined. I will probably know it by tomorrow.

AnnKatF committed:
Merge remote-tracking branch 'origin/master' into catarina/team-structure

# Conflicts:
#	README.md

AliSoftware commented:
Thinking out loud, but couple of other questions / food for thoughts:

* We also don't see the Squad's availability anymore. But tbh I think it's ok, especially now that most of the squads are full and otherwise it changes all the time, not sure it was maintained, and not sure it's useful anymore now that the hiring period has slowed down. (So I'm ok with ditching them personally)
* GitHub handle: Wondering if we could find a way to make the GitHub handle visible/readable? 🤔 
  * On one hand, it's useful to visually see it here to let our brains remember them and see what to type when we want to @ someone in a PR or add someone as PR reviewer. Maybe `[:octocat: @AliSoftware](https://github.com/AliSoftware)` instead?
  * otoh with the current rendering, we see "GitHub" as a link to the profile, but have to click on it to see the handle... but at least the list of people is consistent with all links named the same

AnnKatF commented:
> it's useful to visually see it here to let our brains remember them and see what to type when we want to @ someone in a PR or add someone as PR reviewer

Yes I agree it is useful. I will make this change and have it as you suggested [🐦 <Twitter handle>] [:octocat: <GitHub handle> ]

AnnKatF committed:
Make the handles visible.

AnnKatF committed:
better without the dots?

AnnKatF committed:
better with \|

AnnKatF committed:
adding extra space

AliSoftware commented:
@AnnKatF Alternative table presentation suggestion, lmk what you think:

```
<table>
<thead><th>Squad</th><th>iOS Dev</th><th>Twitter</th><th>GitHub</th></tr>
<tr>
  <td rowspan=3 valign="top"><b>Squad Name</b><br/>Squad Description</td>
  <td>Dev1</td><td><code>@dev1</code></td><td><code>@dev1-babylon</code></td></tr>
  <td>Dev2</td><td><code>@dev2</code></td><td><code>@dev2-babylon</code></td></tr>
  <td>Dev3</td><td><code>@dev3</code></td><td><code>@dev3-babylon</code></td></tr>
</tr>
</table>
```

<table>
<thead><th>Squad</th><th>iOS Dev</th><th>Twitter</th><th>GitHub</th></tr>
<tr>
  <td rowspan=3 valign="top"><b>Squad Name</b><br/>Squad Description</td>
  <td>Dev1</td><td><code>@dev1</code></td><td><code>@dev1-babylon</code></td></tr>
  <td>Dev2</td><td><code>@dev2</code></td><td><code>@dev2-babylon</code></td></tr>
  <td>Dev3</td><td><code>@dev3</code></td><td><code>@dev3-babylon</code></td></tr>
</tr>
</table>

AnnKatF commented:
@AliSoftware looks neat! I like it 🙂 Will make these changes.

AliSoftware commented on README.md:
```diff
@@ -8,85 +8,42 @@ iOS Playbook 📚
+| **Chapter Leads** | Ana Catarina Figueiredo &nbsp;[🐦 @AnnKatFig](https://twitter.com/AnnKatFig) \| [:octocat: @AnnKatF](https://github.com/AnnKatF)  <br>  David Rodrigues &nbsp;[🐦 @dmcrodrigues](https://twitter.com/dmcrodrigues) \| [:octocat: @dmcrodrigues](https://github.com/dmcrodrigues) |
+| **Squad Tech Leads** | Michael Brown &nbsp;[🐦 @mluisbrown](https://twitter.com/mluisbrown) \| [:octocat: @mluisbrown](https://github.com/mluisbrown) |
+
+<br>
```
```suggestion
----

We're organised in Squads. Each squad is composed of iOS, Android and QA Engineers, and a Delivery Manager.

The table below only list the squads in which our iOS Engineers take part, and only list the iOS developers in those respective squads.
```

AnnKatF replied:
What about:

```suggestion
We're organised in Squads. Each squad can be composed of people from Engineering(iOS, Android, Web, Backend), Design and Product.

The table below lists the iOS Engineers per squad.
```

AliSoftware commented:
Actually just thought it would be nice to generate that from an input easier to edit, so I made this:

<details>
<summary>squads.yml</summary>

```yml
- name: Booking
  desc: Face to face appointment booking.
  devs:
    - name: Witold Skibniewski 
      github: mr-v
- name: Consultation
  desc: Flows for all the consultation like prescriptions and video consultation.
  devs:
    - name: Adrian Śliwa
      twitter: adiki91
      github: adiki
    - name: Chitra Kotwani
      twitter: chitrakotwani
      github: chitrakotwani
- name: Core Experience
  desc: General user experience in the app.
  devs:
    - name: Emese Toth
      github: emeseuk
    - name: Giorgos Tsiapaliokas
      github: gtsiap
    - name: Sergey Shulga
      twitter: SergDort
      github: sergdort
    - name: Yuri Karabatov
      twitter: karabatov
      github: karabatov
- name: Native Apps Platform
  desc: Engineering work like tooling, CI and development processes.
  devs: 
    - name: David Rodrigues
      github: dmcrodrigues
      twitter: dmcrodrigues
    - name: Ilya Puchka
      github: ilyapuchka
      twitter: ilyapuchka
    - name: Yasuhiro Inami
      github: inamiy
      twitter: inamiy
    - name: Martin Nygren
      github: zzcgumn
    - name:  Olivier Halligon
      github: AliSoftware
      twitter: aligatr
- name: GP @ Hand
  desc: End to end journey for NHS registration.
  devs: 
    - name: James Birtwell
      github: jimmybee 
- name: Healthcheck
  desc: Overview of your health using a 3D body model (avatar).
  devs: 
    - name: Ben Henshall
      github: Ben-Henshall
      twitter: ben_henshall
    - name: Julien Ducret
      github: brocoo 
- name: Monitor
  desc: Monitoring health metrics like activity, blood, urine, ...
  devs: 
    - name: Anders Ha
      github: andersio
      twitter: _andersha
    - name: Daniel Haight
      github: Daniel1of1
    - name: Daniel Spindelbauer
      github: sdaniel55
      twitter: sdaniel55
    - name: Diego  Petrucci
      github: diegopetrucci
      twitter: diegopetrucci
    - name: Joshua Simmons
      github: j531 
- name: Prescriptions
  desc: Precriptions functionality.
  devs: 
    - name: Adam Borek
      github: TheAdamBorek
      twitter: TheAdamBorek
    - name: Konrad Muchowicz
      github: konrad-em 
- name: Professional Services
  devs: 
    - name: Simon Cass
      github: scass91
      twitter: codercass
- name: SDK
  desc: Develop and mantain the SDK frameworks.
  devs: 
    - name: Viorel Mihalache
      github: viorel15
      twitter: viorelMO 
- name: Tenancy & Features
  devs: 
    - name: Anil Puttabuddhi
      github: anilputtabuddhi
- name: Test Kits
  desc: Managing everything related to Babylon do-at-home tests.
  devs: 
    - name: Michał Kwiecień
      github: MichalTKwiecien
      twitter: kwiecien_co
- name: Triage
  desc: Chatbot functionality.
  devs: 
    - name: Danilo Aliberti
      github: daniloaliberti
    - name: Michael Brown
      github: mluisbrown
      twitter: mluisbrown 
- name: US Professional Services
  desc: Features for app in the US.
  devs: 
    - name: Greg Bryant
    - name: Patrick Westmeyer
      github: bh-pwestmeyer
    - name: Sam Francis
      github: SamFrancis-Babylon
```

</details>

<details>
<summary>squads.rb</summary>

```ruby
#!/usr/bin/env ruby

require 'yaml'

def row(squad)
  squad_info = "<td rowspan='#{squad['devs'].count}' valign='top'><strong>#{squad['name']}</strong><br/>#{squad['desc']}</td>"
  devs_rows = squad['devs'].map do |dev|
    github = dev['github'].nil? ? '' : "@#{dev['github'].strip}"
    twitter = dev['twitter'].nil? ? '' : "<a href='https://twitter.com/#{dev['twitter'].strip}'>@#{dev['twitter'].strip}</a>"
    "<td>#{dev['name']}</td><td>#{github}</td><td>#{twitter}</td>"
  end
  devs_rows[0] = squad_info + "\n      " + devs_rows[0]
  return devs_rows.map { |row| "  <tr>#{row}</tr>" }.join("\n    ")
end

def table(squads)
  rows = squads.map { |squad| row(squad) }.join("\n  ")
  return <<~TABLE
    <!-- DO NOT EDIT MANUALLY: This table has been generated from squads.yml -->
    <!-- Please update squads.yml and run squad.rb to regenerate -->

    <table>
      <thead><th>Squad</th><th>iOS Dev</th><th>GitHub</th><th>Twitter</th></thead>
      #{rows}
    </table>
  TABLE
end

squads = YAML.load_file('squads.yml')
puts table(squads)
```

</details>

<details>
<summary>Result of running `squads.rb`: the HTML table to copy/paste in the README</summary>

```
<!-- DO NOT EDIT MANUALLY: This table has been generated from squads.yml -->
<!-- Please update squads.yml and run squad.rb to regenerate -->

<table>
  <thead><th>Squad</th><th>iOS Dev</th><th>GitHub</th><th>Twitter</th></thead>
    <tr><td rowspan='1' valign='top'><strong>Booking</strong><br/>Face to face appointment booking.</td>
      <td>Witold Skibniewski</td><td>@mr-v</td><td></td></tr>
    <tr><td rowspan='2' valign='top'><strong>Consultation</strong><br/>Flows for all the consultation like prescriptions and video consultation.</td>
      <td>Adrian Śliwa</td><td>@adiki</td><td><a href='https://twitter.com/adiki91'>@adiki91</a></td></tr>
      <tr><td>Chitra Kotwani</td><td>@chitrakotwani</td><td><a href='https://twitter.com/chitrakotwani'>@chitrakotwani</a></td></tr>
    <tr><td rowspan='4' valign='top'><strong>Core Experience</strong><br/>General user experience in the app.</td>
      <td>Emese Toth</td><td>@emeseuk</td><td></td></tr>
      <tr><td>Giorgos Tsiapaliokas</td><td>@gtsiap</td><td></td></tr>
      <tr><td>Sergey Shulga</td><td>@sergdort</td><td><a href='https://twitter.com/SergDort'>@SergDort</a></td></tr>
      <tr><td>Yuri Karabatov</td><td>@karabatov</td><td><a href='https://twitter.com/karabatov'>@karabatov</a></td></tr>
    <tr><td rowspan='5' valign='top'><strong>Native Apps Platform</strong><br/>Engineering work like tooling, CI and development processes.</td>
      <td>David Rodrigues</td><td>@dmcrodrigues</td><td><a href='https://twitter.com/dmcrodrigues'>@dmcrodrigues</a></td></tr>
      <tr><td>Ilya Puchka</td><td>@ilyapuchka</td><td><a href='https://twitter.com/ilyapuchka'>@ilyapuchka</a></td></tr>
      <tr><td>Yasuhiro Inami</td><td>@inamiy</td><td><a href='https://twitter.com/inamiy'>@inamiy</a></td></tr>
      <tr><td>Martin Nygren</td><td>@zzcgumn</td><td></td></tr>
      <tr><td>Olivier Halligon</td><td>@AliSoftware</td><td><a href='https://twitter.com/aligatr'>@aligatr</a></td></tr>
    <tr><td rowspan='1' valign='top'><strong>GP @ Hand</strong><br/>End to end journey for NHS registration.</td>
      <td>James Birtwell</td><td>@jimmybee</td><td></td></tr>
    <tr><td rowspan='2' valign='top'><strong>Healthcheck</strong><br/>Overview of your health using a 3D body model (avatar).</td>
      <td>Ben Henshall</td><td>@Ben-Henshall</td><td><a href='https://twitter.com/ben_henshall'>@ben_henshall</a></td></tr>
      <tr><td>Julien Ducret</td><td>@brocoo</td><td></td></tr>
    <tr><td rowspan='5' valign='top'><strong>Monitor</strong><br/>Monitoring health metrics like activity, blood, urine, ...</td>
      <td>Anders Ha</td><td>@andersio</td><td><a href='https://twitter.com/_andersha'>@_andersha</a></td></tr>
      <tr><td>Daniel Haight</td><td>@Daniel1of1</td><td></td></tr>
      <tr><td>Daniel Spindelbauer</td><td>@sdaniel55</td><td><a href='https://twitter.com/sdaniel55'>@sdaniel55</a></td></tr>
      <tr><td>Diego  Petrucci</td><td>@diegopetrucci</td><td><a href='https://twitter.com/diegopetrucci'>@diegopetrucci</a></td></tr>
      <tr><td>Joshua Simmons</td><td>@j531</td><td></td></tr>
    <tr><td rowspan='2' valign='top'><strong>Prescriptions</strong><br/>Precriptions functionality.</td>
      <td>Adam Borek</td><td>@TheAdamBorek</td><td><a href='https://twitter.com/TheAdamBorek'>@TheAdamBorek</a></td></tr>
      <tr><td>Konrad Muchowicz</td><td>@konrad-em</td><td></td></tr>
    <tr><td rowspan='1' valign='top'><strong>Professional Services</strong><br/></td>
      <td>Simon Cass</td><td>@scass91</td><td><a href='https://twitter.com/codercass'>@codercass</a></td></tr>
    <tr><td rowspan='1' valign='top'><strong>SDK</strong><br/>Develop and mantain the SDK frameworks.</td>
      <td>Viorel Mihalache</td><td>@viorel15</td><td><a href='https://twitter.com/viorelMO'>@viorelMO</a></td></tr>
    <tr><td rowspan='1' valign='top'><strong>Tenancy & Features</strong><br/></td>
      <td>Anil Puttabuddhi</td><td>@anilputtabuddhi</td><td></td></tr>
    <tr><td rowspan='1' valign='top'><strong>Test Kits</strong><br/>Managing everything related to Babylon do-at-home tests.</td>
      <td>Michał Kwiecień</td><td>@MichalTKwiecien</td><td><a href='https://twitter.com/kwiecien_co'>@kwiecien_co</a></td></tr>
    <tr><td rowspan='2' valign='top'><strong>Triage</strong><br/>Chatbot functionality.</td>
      <td>Danilo Aliberti</td><td>@daniloaliberti</td><td></td></tr>
      <tr><td>Michael Brown</td><td>@mluisbrown</td><td><a href='https://twitter.com/mluisbrown'>@mluisbrown</a></td></tr>
    <tr><td rowspan='3' valign='top'><strong>US Professional Services</strong><br/>Features for app in the US.</td>
      <td>Greg Bryant</td><td></td><td></td></tr>
      <tr><td>Patrick Westmeyer</td><td>@bh-pwestmeyer</td><td></td></tr>
      <tr><td>Sam Francis</td><td>@SamFrancis-Babylon</td><td></td></tr>
</table>
```

</details>

Maybe we could even keep `squads.yml` and `squads.rb` somewhere in the repo btw. And next time we need to update the squad list, update the `YML` and regenerate the table, which will automatically adjust everything, the proper `rowspan`, the formatting, and all.

PS: Please double-check the content of the `squads.yml` since I quickly made it via copy/paste and search/replace from the original table but maybe some search/replace went wrong or I removed some items in the process 😅

Commit:
Reformat table using YML and ruby script

Commit:
Add link to Wall-E, add maintainer to Stevenson

Commit:
Fix typo in squad description

Commit:
Use YAML anchors/references to avoid repeating entries in the YAML

Commit:
Improve script readability

Commit:
Wording

AnnKatF commented on README.md:
```diff
@@ -8,91 +8,104 @@ iOS Playbook 📚
 
-By definition, members work on their respective squad, although they are free to work in different squads if the work load justifies it.
+<table>
+  <thead><th>Role</th><th>Person</th><th>GitHub</th><th>Twitter</th></thead>
```
```suggestion
  <thead><th>Role</th><th>Engineer</th><th>GitHub</th><th>Twitter</th></thead>
```

AnnKatF commented on README.md:
```diff
@@ -8,91 +8,104 @@ iOS Playbook 📚
+-->
 
+<table>
+  <thead><th>Squad</th><th>Person</th><th>GitHub</th><th>Twitter</th></thead>
```
```suggestion
  <thead><th>Squad</th><th>Engineer</th><th>GitHub</th><th>Twitter</th></thead>
```

AnnKatF reviewed (commented):
Just two tiny changes.

AnnKatF commented:
This looks good 👍
Good job!

Commit:
Wording

AnnKatF committed:
Update README.md

AnnKatF committed:
Update README.md

Commit:
Simplify script a lot by using comment fences

Allows us to modify the rest of the README around those tables without having the script affect them

Commit:
Update table in script to reflect @AnnKatF changes

AliSoftware commented on scripts/squads.yml:
```diff
@@ -0,0 +1,216 @@
+    desc: Develop and maintain the SDK frameworks.
+    members: 
+      - *Viorel
+  - name: Tenancy & Features
```
@anilputtabuddhi Would be nice if we had a description of this Squad like the others. We have to admit we don't really know how to describe this one 😉

AliSoftware commented on scripts/squads.yml:
```diff
@@ -0,0 +1,216 @@
+    members: 
+      - *Adam
+      - *Konrad
+  - name: Professional Services
```
@scass91 Would be nice if we had a description of this Squad like the others. We have to admit we don't really know how to describe this one 😉

Commit:
typo in script name in comment

Commit:
avoid repeating markers in code

AnnKatF committed:
Squad renaming.

AnnKatF committed:
updated squad name and description.

AnnKatF committed:
updated a squad description.

AliSoftware commented on scripts/squads.yml:
```diff
@@ -0,0 +1,218 @@
+# List of the Squads and their members
+#
+squads:
+  - name: Booking
```
```suggestion
  - name: Appointments
```
@mr-v right?

mr-v replied:
👍

AliSoftware commented on scripts/squads.yml:
```diff
@@ -0,0 +1,218 @@
+      - *Inamiy
+      - *Martin
+      - *Olivier
+  - name: GP @ Hand
```
```suggestion
  - name: GP @ Hand Registrations
```
right @Jimmybee ?

karabatov commented on scripts/squads.yml:
```diff
@@ -0,0 +1,218 @@
+    name: Ilya Puchka
+    github: ilyapuchka
+    twitter: ilyapuchka
+  - &Inamiy
```
```suggestion
  - &Inami
```
?

karabatov commented on scripts/squads.yml:
```diff
@@ -0,0 +1,218 @@
+    members:
+      - *Adrian
+      - *Chitra
+  - name: Onboarding and navigation
```
```suggestion
  - name: Onboarding and Navigation
```

Commit:
Update some squad names

karabatov commented on scripts/squads.yml:
```diff
@@ -0,0 +1,218 @@
+  - &Catarina
+    name: Ana Catarina Figueiredo
+    github: AnnKatF
+    twitter: AnnKatFig
```
```suggestion
    twitter: AnnKatFig
  - &Chesley
     name: Chesley Stephens
     github: cstephens-babylon
```

karabatov commented on scripts/squads.yml:
```diff
@@ -0,0 +1,218 @@
+  - name: US Professional Services
+    desc: Features for app in the US.
+    members: 
+      - *Greg
```
```suggestion
      - *Chesley
      - *Greg
```

AliSoftware replied:
Btw @karabatov any idea who Greg is? We don't have either his GitHub handle nor his twitter and I can't find anyone named Greg on our slack channel...

karabatov replied:
I was under the impression that it's Greg Bryant the engineering manager for US? 🤔

AliSoftware replied:
Well yes it's supposed to be Greg Bryant: https://github.com/Babylonpartners/ios-playbook/pull/225/files#diff-c190676900d06dacf5d18fb8436459c2R59

But if he's a manager, not sure why he's in this list. No other squad list the Engineering manager (nor the DM nor Android devs nor QA, for that matter, since it's the ios-playbook), so not sure why he's in that list then?

AnnKatF replied:
Ok we can remove it then.

karabatov replied:
I've no idea :) He's not contributing directly, that's for sure.

Commit:
Adding @NSMyself back to the list

Commit:
Added Chesley, removed Greg

Commit:
Nitpicking typos

mr-v commented on scripts/squads.yml:
```diff
@@ -0,0 +1,224 @@
+#
+squads:
+  - name: Appointments
+    desc: Face to face appointment booking.
```
```suggestion
    desc: Appointment booking.
```

mr-v commented on README.md:
```diff
@@ -8,91 +8,107 @@ iOS Playbook 📚
 
+<table>
+  <thead><th>Squad</th><th>Engineer</th><th>GitHub</th><th>Twitter</th></thead>
+    <tr><td rowspan='1' valign='top'><strong>Appointments</strong><br/>Face to face appointment booking.</td>
```
```suggestion
    <tr><td rowspan='1' valign='top'><strong>Appointments</strong><br/>Appointment booking.</td>
```

mr-v reviewed (commented):
Appointments squad is not just for face-to-face. Left suggestions for a quick fix.

mr-v committed:
Update description of Appointments squad

jtomanik commented on scripts/squads.yml:
```diff
@@ -0,0 +1,224 @@
+    name: Yasuhiro Inami
+    github: inamiy
+    twitter: inamiy
+  - &James
```
```suggestion
  - &Jakub
    name: Jakub Tomanik
    github: jtomanik
    - &James
```

karabatov commented on scripts/squads.yml:
```diff
@@ -0,0 +1,224 @@
+      - *Diego
+      - *Joshua
+  - name: Prescriptions
+    desc: Precriptions functionality.
```
```suggestion
    desc: Prescriptions functionality.
```

jtomanik commented on scripts/squads.yml:
```diff
@@ -0,0 +1,224 @@
+    members:
+      - *Ben
+      - *Julien
+  - name: Monitor
```
```suggestion
  - name: Identity
    desc: Authentication and Authorization related work
    members:
      - *Jakub
  - name: Monitor
```

karabatov commented on README.md:
```diff
@@ -8,91 +8,107 @@ iOS Playbook 📚
-## 2. Which squad each individual belongs to, and its availability.
+<!-- begin:roles -->
+<!--
+  DO NOT EDIT MANUALLY: This table has been auto-generated.
```
Maybe we can add a pre-commit hook or something?

AliSoftware replied:
It would be nice. Problem is, commit hooks are local, they are not committed and shared across the team. So each person in the team needs to install them on their machine individually

AnnKatF replied:
Thanks for the suggestion!
I will investigate this and see if it is possible to trigger it only for a change in a specific file.

AliSoftware replied:
If pre-commit hooks are indeed local, the only way to check would be to add some CI. But adding CircleCI on the playbook would mean add more stuff to the CircleCI queue which is already quite under pressure; Maybe it would be the occasion to try GitHub Actions (especially since, for the playbook, we could use Linux containers, no need for macOS VMs) to run Danger on it? 🤔

AnnKatF replied:
> commit hooks are local
😕 I wasn't aware of this.

AliSoftware replied:
Well it might still be an idea to investigate as there are suggestions to share hooks accross the team: https://www.viget.com/articles/two-ways-to-share-git-hooks-with-your-team/

But that still require all members in the team to update their configuration. Which we could imagine automate via the `ios-onboarding` bash script we already have (see our [New Hires Checklist](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/NewHiresCheckList.md#setup-guide)) – and ask existing team members to update their config manually once and for all too

karabatov replied:
I just foresee everyone forgetting to update the generated files %) Yeah, not very good that commit hooks are local.

Playbook is open source though so we could run e.g. free Travis on it, no?

AliSoftware replied:
I just realize that even if we share git hooks across teams (using the onboarding script to install on everyone's laptop for example) it still won't perform any check for changes made directly via GitHub suggestions for example. So it's not probably an option anymore even if we managed to sync them on everybody's machine.

yeah CI is the best option. CircleCI has free plans for OSS repos too (that's where all my OSS repos are) so maybe that's the best route to go? I'd prefer CircleCI (to keep what we already use elsewhere and not have to learn 2 different syntaxes depending on repos) or GitHub Actions (since it's new and would be a nice occasion to test it and see if it could be an option even for the main repo in the future) than travis-ci tbh

karabatov replied:
I only said Travis because I know 100% it supports open source repos for free. Sure it's better to use either CircleCI or try out Actions.

This is just a use case that's begging for CI.

chitrakotwani commented on scripts/squads.yml:
```diff
@@ -0,0 +1,224 @@
+    members:
+      - *Witold
+  - name: Consultation
+    desc: Flows for all the consultation like prescriptions and video consultation.
```
There is a separate squad for prescriptions now. Even that is not under our purview anymore. Consultation experience takes care of audio/ video consultation, consultation feedback, pre-consultation work like getting the user ready for consultation, informing about lateness, educating the patient about what to expect and how to be prepared for consultation. Lot of it is planned work

AliSoftware replied:
@chitrakotwani how is this split then? Adrian in Consultation and you in Prescriptions? You both in Prescriptions and nobody from iOS in Consultation anymore?
@AnnKatF can I let you update the yml accordingly?

chitrakotwani replied:
Both Adrian and I are in consultations. Adam and Konrad are in prescriptions. This is mentioned on this page BTW

AnnKatF replied:
I will proceed with the changes.

AliSoftware replied:
Oh ok so the 2 squads themselves and their respective members are correct, only the squad description for Consultations is to be updated, right :+1:

chitrakotwani replied:
Sorry for the confusion 😞 Let me just add suggestion instead

adiki reviewed (approved):
Besides @chitrakotwani comment my part is fine

Commit:
Apply suggestions from code review

Add Identity Squad with @jtomanik

Co-Authored-By: Jakub Tomanik <4947911+jtomanik@users.noreply.github.com>
Co-Authored-By: Yuri Karabatov <yuri.karabatov@babylonhealth.com>

chitrakotwani commented on scripts/squads.yml:
```diff
@@ -0,0 +1,231 @@
+    members:
+      - *Witold
+  - name: Consultation
+    desc: Flows for all the consultation like prescriptions and video consultation.
```
```suggestion
    desc: Consultation experience related features including, but not limited to - audio/video call and replay quality, consultation feedback, pre-consultation preparation and reminders for consultation 
```

Commit:
Update Consultation squad description

Co-Authored-By: Chitra Kotwani <Chitra.kotwani@babylonhealth.com>

Commit:
Merge branch 'master' into catarina/team-structure

# Conflicts:
#	README.md

Commit:
Run script to update table from yml
